Makefile

```diff
@@ -21,11 +21,9 @@ libnavigation-android \
 libnavui-speedlimit \
 libnavui-shield \
 libnavui-status \
-
-NOT_RELEASED_UI_MODULES = \
```

Comment:
adding this back here 😄  https://github.com/mapbox/mapbox-navigation-android/pull/5704